Microsoft.Toolkit.Uwp.DeveloperTools/Microsoft.Toolkit.Uwp.DeveloperTools.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>This library provides XAML user controls and services to help developer building their app. It is part of the UWP Community Toolkit.</Description>
     <PackageTags>UWP Toolkit Windows Controls XAML Developer Tools Accessibility</PackageTags>
     <UseUwpMetaPackage>true</UseUwpMetaPackage>
+    <Configurations>Debug;Release</Configurations>
```

Comment:
Please remove this Configuration tag on project where it's added - it's not relevant to this PR